lua/starfall/instance.lua

```diff
@@ -481,15 +481,15 @@ function SF.Instance:runWithOps(func, ...)
 
 	local prevHook, mask, count = dgethook()
 	local prev = SF.runningOps
-	SF.runningOps = true
+	SF.runningOps = self
 	SF.OnRunningOps(true)
 	dsethook(cpuCheck, "", 2000)
 	self.stackn = self.stackn + 1
 	local tbl = { xpcall(func, xpcall_callback, ...) }
 	self.stackn = self.stackn - 1
 	dsethook(prevHook, mask, count)
 	SF.runningOps = prev
-	SF.OnRunningOps(prev)
```

Comment:
I think we can pass the instance to OnRunningOps. No need to 'not not' and SF.OnRunningOps(true) can be SF.OnRunningOps(self).